### ISSUE TITLE
Local Install Error

### ISSUE THREAD

**Issue by wadethestealth** [atom/github#2296]
I am trying to setup this package from my fork of it, after cloning the fork to my local machine and running `apm install` and `apm link`

I get this error on boot
```
The module '\\?\C:\Users\Wade\Documents\GitHub\github\node_modules\superstring\build\Release\superstring.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 64. This version of Node.js requires
NODE_MODULE_VERSION 69. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
```
```
Error: The module '\\?\C:\Users\Wade\Documents\GitHub\github\node_modules\superstring\build\Release\superstring.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 64. This version of Node.js requires
NODE_MODULE_VERSION 69. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:160:31)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:722:18)
    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:160:31)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (file:///C:/Users/Wade/AppData/Local/atom-nightly/app-1.42.0-nightly8/resources/app.asar/static/index.js:72:46)
    at require (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:146745)
    at Object.<anonymous> (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app.asar\node_modules\text-buffer\node_modules\superstring\index.js:118:15)
    at Object.<anonymous> (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app.asar\node_modules\text-buffer\node_modules\superstring\index.js:357:3)
    at Module.get_Module._compile (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:147429)
    at Object.value [as .js] (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (file:///C:/Users/Wade/AppData/Local/atom-nightly/app-1.42.0-nightly8/resources/app.asar/static/index.js:72:46)
    at require (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:146745)
    at Object.<anonymous> (C:\Users\Wade\Documents\GitHub\github\node_modules\whats-my-line\dist\index.js:4:23)
    at Object.<anonymous> (C:\Users\Wade\Documents\GitHub\github\node_modules\whats-my-line\dist\index.js:114:3)
    at Module.get_Module._compile (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:147429)
    at Object.value [as .js] (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (file:///C:/Users/Wade/AppData/Local/atom-nightly/app-1.42.0-nightly8/resources/app.asar/static/index.js:72:46)
    at require (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:146745)
    at Object.<anonymous> (C:\Users\Wade\Documents\GitHub\github/comment-positioning-container.js:6:1)
    at Object.<anonymous> (C:\Users\Wade\Documents\GitHub\github/comment-positioning-container.js:117:23)
    at Module.get_Module._compile (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:147429)
    at Object.value [as .js] (C:\Users\Wade\AppData\Local\atom-nightly\app-1.42.0-nightly8\resources\app\static\<embedded>:11:150977)
``

Running npm rebuild and re-installing `superstring` does nothing, how do I rebuild this correctly?

**Comment by smashwilson** [comment]
This is a sign that `apm` is building native dependencies for a different version of Electron than your Atom installation is using.

I see that you're using `atom-nightly`. Try rebuilding your dependencies from scratch by running `apm-nightly ci`. (And if you have some kind of aliasing or scripts to launch `atom-nightly` as `atom`, like I do, make sure you do the same to run `apm-nightly` as `apm` :smile:)

**Comment by wadethestealth** [comment]
I realize now that I was running `apm` commands as nightly commands without an alias. I have now added `atom-nightly` to my path and rerunning `apm-nightly install`. I then got a different error relating to build tools. It said I required v140, which i thought was weird since I have v141, but I followed [this answer](https://stackoverflow.com/a/54939414) to install exactly v140 and it built correctly.

Now, the git package and github package don't seem to be recognizing any repositories when I open a workspace.
